salt/pkgrepo/ubuntu/init.sls

```diff
@@ -1,3 +1,16 @@
+legacy-saltstack-pkgrepo:
```

Comment:
I guess if we wanted to make this opt-in we could just wrap it in jinja tied to a pillar value (which we should put in pillar.example) that would require people to set a pillar value in order to have the old one removed.